packages/generator-single-spa/src/vue/generator-single-spa-vue.js

```diff
@@ -35,13 +37,28 @@ module.exports = class SingleSpaVueGenerator extends Generator {
     } else if (status !== 0) {
       process.exit(status);
     } else {
+      // We purposely do not attempt to install in one command using presets to avoid being too restrictive with application configuration
       spawnSync(command, args.concat(["add", "single-spa"]), {
         stdio: "inherit",
-        cwd,
+        cwd: this.cwd,
         env: Object.assign({}, process.env, {
-          VUE_CLI_SKIP_DIRTY_GIT_PROMPT: true
-        })
+          VUE_CLI_SKIP_DIRTY_GIT_PROMPT: true,
+        }),
       });
     }
   }
+
+  async finished() {
+    const usedYarn = this.fs.exists(path.resolve(this.cwd, "yarn.lock"));
```

Comment:
I chose to detect if yarn was used because it seems like it'd a more reliable check. I'm not familiar with other package managers (pnpm, cnpm) but I figure the base `npm` command would be generic enough that users could infer how to update the command themselves.